tests/test_interrupted_sync.py

```diff
@@ -0,0 +1,198 @@
+                    
+                    replication_key = next(iter(expected_replication_keys[stream]))
+                    
+                    interrupted_bookmark_value = interrupted_sync_states['bookmarks'][stream]
+                    if stream in completed_streams:
+                        # Verify at least 1 record was replicated in the second sync
+                        self.assertGreaterEqual(second_sync_count,
+                                            1, 
+                                            msg="Incorrect bookmarking for {0}, at least one or more record should be replicated".format(stream))
+
+                    elif stream == interrupted_sync_states.get('currently_syncing', None):
+                        # For interrupted stream records sync count should be less equals
+                        self.assertLessEqual(second_sync_count,
+                                            first_sync_count,
+                                            msg="For interrupted stream - {0}, seconds sync record count should be lesser or equal to first sync".format(stream))
+
+                    elif stream in pending_streams:
+                        # First sync and second sync record count match
+                        self.assertGreaterEqual(second_sync_count,
+                                                first_sync_count,
+                                                msg="For pending sync streams - {0}, second sync record count should be more than or equal to first sync".format(stream))     
+
+                    else:
+                        raise Exception("Invalid state of stream {0} in interrupted state, please update appropriate state for the stream".format(stream))
+                    
+                    if stream in {'product_reviews', 'product_variants', 'order_fulfillments'}:
+                        repl_key = {'product_reviews': 'product_yotpo_id',
+                                    'order_fulfillments': 'order_id',
+                                    'product_variants': 'yotpo_product_id'
+                                }[stream]
+                        for record in second_sync_messages:
+                            # Verify the second sync replication key value is Greater or Equal to the first sync bookmark for each id
+                            replication_key_value = record.get(replication_key)
+                            repl_key_id = str(record.get(repl_key))
+                            interrupt_bookmark_value = interrupted_bookmark_value[repl_key_id] if repl_key_id in interrupted_bookmark_value.keys() else start_date
+
+                            self.assertLessEqual(interrupt_bookmark_value,
+                                                replication_key_value,
+                                                msg="Interrupt bookmark was set incorrectly, a record with a lesser replication-key value was synced compared to interrupt bookmark value. Record = {}".format(record))
```

Comment:
Can we reduce the assertion error msg length? Same suggestion for other assertion errors as well.

Reply:
Instead of reducing the assertion error message, aligned them properly in multiple lines.